### PR TITLE
Bottlerocket ECS settings documented in TOML DSL

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Contributes to #203

**Description of changes:**

This adds templates and styles to render the DSL as described in #204. It also includes the `settings.ecs.*` and `settings.motd` ported from the bottlerocket-os/bottlerocket README.

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: data/settings/1.14.x/ecs.toml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# tags
+
+[[docs.tag.startup-only]]
+heading = "Startup Only Settings"
+description = """
+These settings affect how the instances joins the ECS cluster.
```
Comment (arnaldo2792) [inline data/settings/1.14.x/ecs.toml:6]:
```suggestion
These settings affect how the instances join the ECS cluster.
```

File: data/settings/1.14.x/ecs.toml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+heading = "Log Settings"
+description = "Settings related to logs from the ECS Agent"
+
+
+# reference
+
+[[docs.ref.cluster]]
+description = """
+The name or [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of your Amazon ECS cluster.
+If left unspecified, Bottlerocket will join your default cluster.
```
Comment (arnaldo2792) [inline data/settings/1.14.x/ecs.toml:24]:
```suggestion
If left unspecified, the Bottlerocket host will join your default cluster.
```

File: data/settings/1.14.x/ecs.toml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+see = [
+    ["`ECS_INSTANCE_ATTRIBUTES` in the [ECS Agent Environment Variables](https://github.com/aws/amazon-ecs-agent/blob/master/README.md#environment-variables)"]
+]
+[[docs.ref.instance-attributes.example]]
+comment = "Example user data for setting up attributes"
+[[docs.ref.instance-attributes.example.multiline]]
+attribute1 = "foo"
+attribute2 = "bar"
+
+[[docs.ref.allow-privileged-containers]]
```
Comment (arnaldo2792) [inline data/settings/1.14.x/ecs.toml:44]: Almost all the settings for ECS have a corresponding environment variable, should we be consistent and add a `see` section that references the corresponding env variable?

File: data/settings/1.14.x/ecs.toml [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+[[docs.ref.metadata-service-rps]]
+description = """
+The steady state rate limit of the throttling configurations set for the task metadata service.
+"""
+note = "This directly maps to the values set by the ECS_TASK_METADATA_RPS_LIMIT environment variable."
+
+[[docs.ref.metadata-service-burst]]
+description = "The burst rate limit of the throttling configurations set for the task metadata service."
+note = "This directly maps to the values set by the ECS_TASK_METADATA_RPS_LIMIT environment variable."
```
Comment (arnaldo2792) [inline data/settings/1.14.x/ecs.toml:117]: Same as in `metadata-service-rps`

File: data/settings/1.14.x/ecs.toml (hunk 3 above)
Comment (stockholmux) [reply data/settings/1.14.x/ecs.toml:44]: Is this the confusingly opposite `ECS_DISABLE_PRIVILEGED`?

Comment (arnaldo2792) [reply data/settings/1.14.x/ecs.toml:44]: Correct!

**Pull request merged**